packages/manfred-common/src/doc-parts/experience-section/experience-section.helpers.ts

```diff
@@ -0,0 +1,15 @@
+import { ExperienceVm } from './experience-section.vm';
+import { dateExtractor, sortByDate } from '../../helpers';
```

Comment:
Use '@/helpers' on import and move it at first line

Reply:
With import in this way I get test failures